packages/cx-layout/src/components/Layout/tokens/cxLayout.ts

```diff
@@ -71,6 +77,7 @@ const Default = asLayoutToken({
     ...Base.Components,
     ...Header.Components,
     ...Footer.Components,
+    ...ConstrainSite.Theme,
```

Comment:
This doens't belong in `Components` domain.  Either add it in `Compose` domain (in which case we should rename it `WithSiteConstraint` (or, probably better, `WithSiteWidthConstaint`), like so:
```
Compose: {
  WithSiteWidthConstraint,
  }
  ``
Or just get rid of this token and put the classes directly in the `Layout` and/or `Spacing` domain of the base token.